extensions/raycast-today/README.md

```diff
@@ -0,0 +1 @@
+# ToDay
```

Comment:
Worth adding some useful information to readme. You can list your features and give a introduce to your extension.

Reply:
Sorry i published a bit too fast and realised I wasn't missing some doc 😅 All fixed